test/devicecontroller/DefaultDeviceController.test.ts

```diff
@@ -2257,21 +2257,10 @@ describe('DefaultDeviceController', () => {
       spy.restore();
     });
 
-    it('cannot connect the video stream to the video element if getUserMedia API fails', async () => {
-      const spy = sinon.spy(DefaultVideoTile, 'connectVideoStreamToVideoElement');
-      await deviceController.chooseVideoInputDevice(stringDeviceId);
-      domMockBehavior.getUserMediaSucceeds = false;
-      deviceController.startVideoPreviewForVideoInput(element);
-      await delay(100);
-      expect(spy.called).to.be.false;
-      spy.restore();
-    });
-
-    it('does not disconnect or connect the video stream for preview if no active video exsits', async () => {
+    it('does not disconnect or connect the video stream for preview if no active video exists', () => {
       const spy1 = sinon.spy(DefaultVideoTile, 'disconnectVideoStreamFromVideoElement');
       const spy2 = sinon.spy(DefaultVideoTile, 'connectVideoStreamToVideoElement');
       deviceController.startVideoPreviewForVideoInput(element);
-      await delay(100);
       expect(spy1.called).to.be.false;
```

Comment:
Can we get a test to prove that it only calls `gUM` once?

Reply:
We remove the getUserMedia call so there would be no call right?

Reply:
There should be a single call when they choose the video input, and no additional call when they start the preview.

This is easy to do with sinon.spy; see elsewhere in the codebase.

Reply:
Not in this `it` block, you can validate that in the previous `it`. In this `it`, you can ensure that `gUM` is not called.

Reply:
Thanks. Updated in both test case to check for getUserMedia.